config/docker/chromeos-tast/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:bullseye
```

Comment:
Do we need full `bullseye` image or could we do with `bullseye-slim` one?

Reply:
Good point, I'll give it a try to find out.

Reply:
Yep that worked, updated.